Scoop/Controller/Onboarding/AboutYouViewController.swift

```diff
@@ -0,0 +1,173 @@
+                      return
+                  }
+            
+            Networking.shared.currentUser.name = name
+            Networking.shared.currentUser.pronouns = pronouns
+            Networking.shared.currentUser.hometown = hometown
+            Networking.shared.currentUser.year = year
+            
+            self.delegate?.didTapNext(navCtrl, nextViewController: nil)
+        }
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Next", image: nil, primaryAction: nextAction, menu: nil)
+        
+        setupStackView()
+    }
+    
+    private func setupStackView() {
+        stackView.axis = .vertical
+        stackView.distribution = .fill
+        stackView.spacing = 20
+        stackView.alignment = .leading
+        view.addSubview(stackView)
+        
+        stackView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(60)
+            make.center.equalToSuperview()
+        }
+        
+        let nameLabel = UILabel()
+        nameLabel.font = .systemFont(ofSize: 22)
+        nameLabel.text = "Name"
+        nameLabel.textColor = .black
+        stackView.addArrangedSubview(nameLabel)
+        
+        nameTextField.font = .systemFont(ofSize: 22)
+        nameTextField.textColor = .darkGray
+        nameTextField.placeholder = "name..."
+        stackView.addArrangedSubview(nameTextField)
+        
+        let pronounsLabel = UILabel()
+        pronounsLabel.font = .systemFont(ofSize: 22)
+        pronounsLabel.text = "Pronouns"
+        pronounsLabel.textColor = .black
+        stackView.addArrangedSubview(pronounsLabel)
+        
+        pronounsPicker.delegate = self
+        pronounsPicker.dataSource = self
+        
+        pronounsTextField.delegate = self
+        pronounsTextField.font = .systemFont(ofSize: 22)
+        pronounsTextField.textColor = .darkGray
+        pronounsTextField.placeholder = "pronouns..."
+        pronounsTextField.inputView = pronounsPicker
+        stackView.addArrangedSubview(pronounsTextField)
+        
+        let hometownLabel = UILabel()
+        hometownLabel.font = .systemFont(ofSize: 22)
+        hometownLabel.text = "Hometown"
+        hometownLabel.textColor = .black
+        stackView.addArrangedSubview(hometownLabel)
+        
+        hometownTextField.font = .systemFont(ofSize: 22)
+        hometownTextField.textColor = .darkGray
+        hometownTextField.placeholder = "hometown..."
+        stackView.addArrangedSubview(hometownTextField)
+        
+        let yearLabel = UILabel()
+        yearLabel.font = .systemFont(ofSize: 22)
+        yearLabel.text = "Class Year"
+        yearLabel.textColor = .black
+        stackView.addArrangedSubview(yearLabel)
+        
+        yearPicker.delegate = self
+        yearPicker.dataSource = self
+        
+        yearTextField.delegate = self
+        yearTextField.font = .systemFont(ofSize: 22)
```

Comment:
maybe create variable to store main font size of 22

Reply:
We're going to refactor fonts and everything once we receive high fidelity from design